tests/unit/igraph_layout_umap.c

```diff
@@ -113,11 +113,14 @@ int main() {
     igraph_destroy(&graph);
 
     printf("Empty graph:\n");
-	igraph_small(&empty_graph, 0, IGRAPH_UNDIRECTED, -1);
+    igraph_small(&empty_graph, 0, IGRAPH_UNDIRECTED, -1);
     IGRAPH_ASSERT(igraph_layout_umap(&empty_graph, NULL, &layout, 0.01, 500, -1) == IGRAPH_SUCCESS);
     igraph_matrix_print(&layout);
-    igraph_destroy(&empty_graph);
 
+    printf("Check error for negative min_dist.\n");
+    CHECK_ERROR(igraph_layout_umap(&empty_graph, NULL, &layout, -0.01, 500, -1), IGRAPH_EINVAL);
```

Comment:
Actually, we don't need to explicitly check for a return code of `IGRAPH_SUCCESS` in most cases. If there is an error, it _should_ be triggered using `IGRAPH_ERROR`, which means that `igraph_error()` will be called, which by default terminated the program with `abort()`, and causes the test to fail. `igraph_abort()` is set up to print a stack trace whenever address sanitizer is enabled, so we can tell where the failure happened.

Checking the error code is most useful when the error handler is changed to something else (e.g. `igraph_error_handler_ignore()`)

Of course, this check does not hurt in any way, and does provide a little bit of security (what if a wrong error code is returned in other ways than `IGRAPH_ERROR`, e.g. a missing `return` statement).